app/scripts/controllers/createPersistentVolumeClaim.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+/**
+ * @ngdoc function
+ * @name openshiftConsole.controller:CreatePersistentVolumeClaimController
+ * @description
+ * # CreatePersistentVolumeClaimController
+ * Controller of the openshiftConsole
+ */
+angular.module('openshiftConsole')
+  .controller('CreatePersistentVolumeClaimController', function ($filter, $routeParams, $scope, $window, ApplicationGenerator, DataService, Navigate, ProjectsService) {
+    $scope.alerts = {};
+    $scope.renderOptions = {
+      hideFilterWidget: true
+    };
+    $scope.projectName = $routeParams.project;
+    $scope.serviceName = $routeParams.service;
+
+    // Prefill route name with the service name.
+    $scope.routing = {
+      name: $scope.serviceName || ""
+    };
+
+    $scope.breadcrumbs = [
+      {
+        title: $scope.projectName,
+        link: "project/" + $scope.projectName
+      },
+      {
+          title: "Storage",
+          link: "project/" + $scope.projectName + "/browse/storage"
+      },
+      {
+        title: "Request Storage"
+      }
+    ];
+
+    ProjectsService
+      .get($routeParams.project)
+      .then(_.spread(function(project, context) {
+        $scope.project = project;
+        // Update project breadcrumb with display name.
+        $scope.breadcrumbs[0].title = $filter('displayName')(project);
+        var labels = {},
+            orderByDisplayName = $filter('orderByDisplayName');
+
+        DataService.list("services", context, function(services) {
```

Comment:
I'm unclear why the controller is requesting services. You shouldn't need a service to create a PVC? (Here and the route params above.)